crates/re_viewer/src/ui/event_log_view.rs

```diff
@@ -175,38 +173,44 @@ fn table_row(
                 path_op.data_ui(ctx, ui, UiVerbosity::All, &query);
             });
         }
-        LogMsg::ArrowMsg(msg) => match MsgBundle::try_from(msg) {
-            Ok(MsgBundle {
-                msg_id,
-                entity_path,
-                time_point,
-                cells: components,
-            }) => {
-                row.col(|ui| {
-                    ctx.msg_id_button(ui, msg_id);
-                });
-                row.col(|ui| {
-                    ui.monospace("ArrowMsg");
-                });
-                for timeline in ctx.log_db.timelines() {
+        LogMsg::ArrowMsg(msg) => match DataTable::try_from(msg) {
```

Comment:
note about this only make sense because we don't batch in practice + this is about to disappear